daml-script/daml/Daml/Script.daml

```diff
@@ -867,15 +867,15 @@ getUser' userId participant = checkUserNotFound userId $ lift $ Free $ GetUser G
   userId
 
 -- | List all users.
-listUsers : Script [User]
-listUsers = listUsers' None
+listAllUsers : Script [User]
```

Comment:
I think it makes sense to extend the docs to mention that we concatenate all results from pagination. Otherwise the name might sound a bit confusing to people.